glide.yaml

```diff
@@ -46,11 +46,12 @@ import:
 - package: github.com/spf13/cobra
   version: 0.0.1
 - package: github.com/spf13/viper
-  version: ^1
+  version: 25b30aa063fc18e48662b86996252eabdcf2f0c7
```

Comment:
I don't think this is necessary, it looks like viper did a bad release 1.0.1 and then rolled back the change and re-released as 1.0.2, so our pin `^1` should still work.